roles/rsyslog/roles/README.md

```diff
@@ -21,7 +21,7 @@ The `defaults` directory includes:
 
 The `tasks` directory includes 2 tasks file:
   - `main.yaml` - tasks for deploying the config files
-    This file is sets `rsyslog_role_packages` and `rsyslog_role_rules` and includes the task that deploys the files.
+    This file is sets `__rsyslog_packages` and `__rsyslog_rules` and includes the task that deploys the files.
```

Comment:
@nhosoi This file sets. I think we can drop the "is".

Reply:
Thanks.  Fixed.